Documentation/kubernetes.md

```diff
@@ -31,7 +31,7 @@ If you want to deploy `flannel` securely in a shared namespace or want more fine
 Other options include [Kyverno](https://kyverno.io/policies/pod-security/) and [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper).
 # Annotations
 
-*  `flannel.alpha.coreos.com/public-ip`, `flannel.alpha.coreos.com/public-ipv6`: Define the used public IP of the node. If configured when Flannel starts it'll be used as the `public-ip` and `public-ipv6` flag.
+*  `flannel.alpha.coreos.com/node-public-ip`, `flannel.alpha.coreos.com/node-public-ipv6`: Define the used public IP of the node. If configured when Flannel starts it'll be used as the `public-ip` and `public-ipv6` flag.
```

Comment:
*  `flannel.alpha.coreos.com/node-public-ip`, `flannel.alpha.coreos.com/node-public-ipv6`: Optionally set by the user, it defines the IP of the node for pod communications, e.g. the vtep of vxlan. The IP must be part of a local interface. Flanneld will use it as value for `flannel.alpha.coreos.com/public-ip` and `flannel.alpha.coreos.com/public-ipv6`

*  `flannel.alpha.coreos.com/public-ip`, `flannel.alpha.coreos.com/public-ipv6`: Set by flanneld informs of the node IP chosen for pod communication

* `flannel.alpha.coreos.com/public-ip-overwrite`, `flannel.alpha.coreos.com/public-ipv6-overwrite` optionally set by the user, defines the IP of the node for pod communications, e.g. the vtep of vxlan. Unlike the `node-public-ip` annotation, the IP does need to be part of a local interface. This annotation should be used if the node is behind a NAT. It can be automatically set to a nodes `ExternalIP` using the [flannel-node-annotator](https://github.com/alvaroaleman/flannel-node-annotator).See also the "NAT" section in [troubleshooting](./troubleshooting.md) if UDP checksums seem corrupted. See also the "NAT" section in [troubleshooting](./troubleshooting.md) if UDP checksums seem corrupted.

Reply:
The NAT UDP checksum issue is not related with the ip-overwrite that it's based on external NAT.